src/enzyme.d.ts

```diff
@@ -2,12 +2,19 @@
 // `@types/enzyme` because that brings in all the React types.
 
 declare module 'enzyme' {
+  import type { PreactElement } from 'preact';
+
   export class ReactWrapper {
     getDOMNode(): HTMLElement;
+    getElements(): PreactElement[];
```

Comment:
From the docs https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/getElements.html

The return type is defined as `ReactElement[]` there, so I guess this is the equivalent, although any array should work here.

Reply:
The docs you are linking to refer to the _shallow_ API, but we are not using the shallow API. We are using the "Full DOM Rendering" API. I don't see a mention of `getElements` for the "full" API. Enzyme's docs are not generated from actual types though, so if this code runs, then I'm assuming this is an omission in the docs.

Reply:
Hhm, good catch. I think I had too many tabs open and got them mixed.

It's funny it's not documented, because the method does work. I saw references to other undocumented methods in GitHub issues, but I wanted to avoid using those.

I'm seeing some inconsistent behaviors with this method when used in a full DOM wrapper, so perhaps that's the reason is not documented and we should in fact not use it.